src/OctoshiftCLI.Tests/gei/Commands/GenerateScriptCommandTests.cs

```diff
@@ -620,6 +621,22 @@ public async Task It_Uses_Ado_Pat_When_Provided()
             mockEnvironmentVariableProvider.VerifyNoOtherCalls();
         }
 
+        [Fact]
+        public void It_Adds_Skip_Releases_To_Migrate_Repo_Command_When_Provided()
+        {
+            var repo = "foo-repo";
+            var repos = new List<string>() { repo };
+
+            var command = new GenerateScriptCommand(new Mock<OctoLogger>().Object, null, null, null);
```

Comment:
```suggestion
            var command = new GenerateScriptCommand(TestHelpers.CreateMock<OctoLogger>().Object, null, null, null);
```

Reply:
Updated in 3ddc2374b698be4f5b2e1a721c85fed6cddf2ec6!